### PR TITLE
Publish legal info about Foundation

### LINKED ISSUE #273: Publish legal info about Foundation (KvK registration number, etc)
This includes:
- [x] de officiële naam van de instelling
- [x] het nummer dat aan de instelling is toegekend bij inschrijving in het handelsregister (RSIN)
- [x] het post- of bezoekadres, dan wel het telefoonnummer, dan wel het e-mailadres van de instelling
- [x] de doelstelling volgens de regelgeving van de instelling
- [x]  de hoofdlijnen van het actuele beleidsplan van de instelling
- [ ] de bestuurssamenstelling, het beloningsbeleid en de namen van de bestuurders

The official government list is here: https://www.belastingdienst.nl/wps/wcm/connect/bldcontentnl/belastingdienst/zakelijk/bijzondere_regelingen/goede_doelen/algemeen_nut_beogende_instellingen/gegevens_van_een_anbi_publiceren_op_een_internetsite/gegevens_van_een_anbi_publiceren_op_een_internetsite

De uitzonderingen zijn niet van toepassing.

### PR DESCRIPTION
**Pull request by ElenaFdR**
Fixes #273

### CODE REVIEW

File: organization/contact-us.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
++ twitter: [@publiccodenet](twitter.com/publiccodenet)
+
+## Communications channels
+
+Our canonical source of truth is this website. Watch us work and join our discussions at:
+
++ github.com/publiccodenet
+
+We publish our news and announcements on:
+> + [our blog](blog.publiccode.net)
```
Comment (bvhme) [inline organization/contact-us.md]: This is in a block quote.

Also, there should be an empty rule before a list.

File: organization/contact-us.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
++ email: <info@publiccode.net>
++ post: Keizersgracht 617, 1017 DS, Amsterdam, the Netherlands
++ telephone: +31 6 17 96 02 05 (Boris van Hoytema's phone)
++ twitter: [@publiccodenet](twitter.com/publiccodenet)
+
+## Communications channels
+
+Our canonical source of truth is this website. Watch us work and join our discussions at:
+
++ github.com/publiccodenet
```
Comment (bvhme) [inline organization/contact-us.md]: Should this be a link?

File: organization/index.md [hunk 3]
```diff
@@ -2,16 +2,34 @@
 type: Index
 ---
 
-# Organization
+# About us
```
Comment (bvhme) [inline organization/index.md]: I find this confusing as in relation to the entire 'about'.

```suggestion
# About the organization
```

File: organization/index.md [hunk 4]
```diff
@@ -2,16 +2,34 @@
...
+## How we're organized
+
+This is how we're building the organization to support our mission:
 
-* [Mission](mission.md)
 * [Roadmap](roadmap.md)
-* [Goals](goals.md)
-* [Cultural values](cultural-values.md)
 * [Milestones](milestones.md)
+* [Legal registration and banking information]
```
Comment (bvhme) [inline organization/index.md]: Missing link

File: organization/index.md [hunk 5]
```diff
@@ -2,16 +2,34 @@
...
 * [Keys to success](keys-to-success.md)
-* [Communication channels](communication-channels.md)
 * [Definition of done](definition-of-done.md)
-* [Governance model](governance-model.md)
+
+## Legal registration and banking information
+
+We're registered as a vereniging (association) under Dutch law. Our full name is the Foundation for Public Code Vereniging. 
+
+[See our association bylaws].
```
Comment (bvhme) [inline organization/index.md]: Missing link

File: organization/index.md [hunk 6]
```diff
@@ -2,16 +2,34 @@
...
+
+Our Kamer van Koophandel registration number is 74996452.
+
+> Our bank number is:
```
Comment (bvhme) [inline organization/index.md]: Block quote with a missing number.

**Comment by ElenaFdR** [comment]
See #273 for information this PR is still missing.

**Comment by ElenaFdR** [comment]
#312 covers the missing info, so I propose merging and closing this PR.

**Pull request closed without merging**